.github/workflows/check_format_code.yml

```diff
@@ -0,0 +1,32 @@
+name: Style Check
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  formatting-check:
+    name: Formatting Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        path:
+          - check: 'stout'
+            exclude: ''
+          - check: 'test'
+            exclude: 'BUILD.bazel'
```

Comment:
Do we need to exclude `BUILD.bazel`? Does clang-format try to check it if we don't?

Reply:
Actually we do not need to exclude `BUILD.bazel` file, cause this action will check the following list of extensions:
`.h , .H, .hpp, .hh, .h++, .hxx`
`.c, .C, .cpp, .cc, .c++, .cxx`
So I will remove this file from the `exclude` section.